common/src/main/resources/tachyon-default.properties

```diff
@@ -85,3 +85,4 @@ tachyon.user.quota.unit.bytes=8MB
 tachyon.user.file.buffer.bytes=1MB
 tachyon.user.remote.block.reader.class=tachyon.client.tcp.TCPRemoteBlockReader
 tachyon.user.remote.read.buffer.size.byte=8MB
+tachyon.user.network.netty.worker.threads=1
```

Comment:
Probably should default to 0 to get the 2x # of cores default provided by netty?